kornia/augmentation/container/augmentation_sequential.py

```diff
@@ -85,17 +85,16 @@ def apply_to_mask(
         return input
 
     def apply_to_bbox(
-        self, input: torch.Tensor, module: nn.Module, param: Optional[Dict[str, torch.Tensor]] = None,
-        mode: str = "xyxy"
+        self,
+        input: torch.Tensor,
+        module: nn.Module,
+        param: Optional[Dict[str, torch.Tensor]] = None,
```

Comment:
In the previous api we were naming this in plural, right ? `params`

Reply:
It has ~~~params~~~ in the forward method. So that for those functions that applies one nn.Module, I used singular to differentiate.